app/UiTPAS/Listeners/SendSlackMessageWhenOrganizerIsRequested.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\UiTPAS\Listeners;
+
+use App\Domain\Integrations\Events\UdbOrganizerAdded;
+use App\Domain\Integrations\IntegrationType;
+use App\Domain\Integrations\Repositories\IntegrationRepository;
+use App\Domain\Integrations\Repositories\UdbOrganizerRepository;
+use App\Notifications\Notifier;
+use App\UiTPAS\Slack\UdbOrganizerMessageBuilder;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Psr\Log\LoggerInterface;
+use Throwable;
+
+final class SendSlackMessageWhenOrganizerIsRequested implements ShouldQueue
```

Comment:
I would make it more general, like `NotifyIntegrationChanged` and change it to something like `NotifyUdbOrganizerAdded`

Reply:
I will got for `NotifyIntegrationRequested`, to communicate this is part of an approval flow.